src/models/Action.js

```diff
@@ -108,7 +109,9 @@ module.exports = function(formio) {
           return;
         }
 
-        hook.alter('actionMachineName', `${form.name}:${document.name}`, document, done);
+        const formMachineName = _last(form.machineName.split(':'));
```

Comment:
I am pretty sure this will break a lot of implementations. Please revise.

Reply:
We've discussed it on the devSup and decided that it's correct.